code/game/machinery/cloning.dm

```diff
@@ -126,6 +126,15 @@
 	if (is_operational() && (!isnull(occupant)) && (occupant.stat != DEAD))
 		to_chat(user, "Current clone cycle is [round(get_completion())]% complete.")
 
+/obj/machinery/clonepod/return_air()
+	// We want to simulate the clone not being in contact with
+	// the atmosphere, so we'll put them in a constant pressure
+	// nitrogen. They'll breathe through the chemicals we pump into them.
+	var/global/datum/gas_mixture/immutable/cloner/GM //global so that there's only one instance made for all cloning pods

```

Comment:
I've said many times that as soon as that FR goes through I will happily support using static over global
I've told you why I disagree with that argument several times and I can't help it if you don't agree yourself
Either way is fiiiine
Show me the official policy on static vs global or shut the fuck up about these pointless semantics